src/client/cli/formatter/table_formatter.cpp

```diff
@@ -135,3 +136,28 @@ std::string mp::TableFormatter::format(const ListReply& reply) const
 
     return fmt::to_string(buf);
 }
+
+std::string mp::TableFormatter::format(const FindReply& reply) const
+{
+    fmt::memory_buffer buf;
+
+    if (reply.images_info().empty())
+        return "No images found.\n";
+
+    fmt::format_to(buf, "{:<24}{:<18}{:<17}{:<}\n", "Image", "Aliases", "Version", "Description");
+
+    for (const auto& image : reply.images_info())
+    {
+        auto aliases = image.aliases_info();
+
+        mp::format::filter_aliases(aliases);
+
+        auto alias = aliases.cbegin();
+        fmt::format_to(buf, "{:<24}{:<18}{:<17}{:<}\n",
+                       (alias->remote_name().empty() ? "" : alias->remote_name() + ":") + alias->alias(),
+                       fmt::format("{}", fmt::join(aliases.cbegin() + 1, aliases.cend(), ",")), image.version(),
+                       fmt::format("{} {}", image.os(), image.release()));
```

Comment:
`snapcraft:*` images apparently have an empty os field, so the description starts with a space here, which breaks the alignment:

```
$ multipass find
Image                   Aliases           Version          Description
snapcraft:core          core16            20190515          Snapcraft builder for Core 16
snapcraft:core18                          20190515          Snapcraft builder for Core 18
core                    core16            20190424         Ubuntu Core 16
core18                                    20190213         Ubuntu Core 18
[...]
```

Do you see the same?

Reply:
yep I see it too

Reply:
Indeed, fixing.